### DISCUSSION TITLE
Welcome to DNS Changer Desktop Discussions!

### DISCUSSION THREAD

**Discussion by sajjadmrx** [DnsChanger/dnsChanger-desktop#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OrchidWarrior** [comment]
Thank you so much for this project. 😇

**Comment by sajjadmrx** [comment]
> Thank you so much for this project. 😇

🙏💚

**Comment by solonoturningpoint** [comment]
can you add a feature where it automatically switches to the fastest dns everytime the current dns is unideal? if so how long would it take for you to add the feature

**Comment by rekt0ro** [comment]
درود
طراحی و عملکرد برنامه خیلی خوبه فقط اگه بعد از دیسکانکت کردنش دی ان اس قبلی که خودمون دستی زدیم رو خودکار بر میگردوند عالی میشه.

به عنوان مثال: من الان به صورت دستی دی ان اس گوگل رو ست کردم روی ویندوز و میخوام با این برنامه دی ان اس رو سویچ کنم روی الکترو و بعد از دیسکانکت کردنش برنامه اتوماتیک همون دی ان اس گوگل که خودم دستی ست کرده بودم رو برگردونه! به عبارتی وی پی ان طور بشه!
مرسی.

**Reply by sajjadmrx** [reply]
https://github.com/DnsChanger/dnsChanger-desktop/discussions/31#discussioncomment-9804156

**Comment by blackvoidx** [comment]
سلام
خسته نباشی
یه موضوعی که هست و خواستم مطرح کنم اینه که وقتی بین تب های برنامه سوییچ میشه
برنامه از حالت connected خارج میشه و مجددا باید کانکت کرد.
دوم اینکه وقتی برنامه باز میشه دیفالت یه dns ست رو نشون میده که dns خود سیستم هست با عنوان unknown هم نشون میده
بعد connected و سپس Unconnect دیگه اون dns دیفالت ست نمیشه اگه میشد که اینم اضافه شه و باز به dns خود سیستم برگرده بهتر میشد

در آخر هم دمت گرم بابت این پروژه👍💚

**Reply by blackvoidx** [reply]
اینم اضافه کنم که شاید کمک کنه
من روی linux mint 21 نسخه mate نصب کردم.

**Reply by sajjadmrx** [reply]
>  از حالت connected خارج میشه و مجددا باید کانکت کرد.

سلام، شما وقتی دکمه connect رو میزنید در اصل DNS رو ست میکنید،  
پس وقتی بین تب ها سوییچ میکنید یا حتی برنامه رو ببندید اکشن خاصی اتفاق نمیفته و نمایش "connected" بیشتر جنبه UI داره


 > دوم اینکه وقتی برنامه باز میشه دیفالت یه dns ست رو نشون میده که dns خود سیستم هست با عنوان unknown هم نشون میده
بعد connected و سپس Unconnect دیگه اون dns دیفالت ست نمیشه اگه میشد که اینم اضافه شه و باز به dns خود سیستم برگرده بهتر میشد

بله یکی از دوستان هم قبلا این پیشنهاد رو داده بودن، که به برنامه اضافه شده
قسمت اضافه کردن Server و تب "default" اونجا اون سروری که لازمه بعد از "Unconnect" ست بشه رو وارد کنید:

![image](https://github.com/DnsChanger/dnsChanger-desktop/assets/66132114/2e926d09-39dc-4389-93b3-99730d44b93e)


> در آخر هم دمت گرم بابت این پروژه👍💚

ممنون بابت انگیزه دادنتون 🙏🫡

**Reply by blackvoidx** [reply]
عالی درود🙏💚

**Comment by FraKOne** [comment]
Hello and many tanx for this useful app. I don't find a Mac intel 2.3.2 version, is it normal ?

**Comment by Unique168** [comment]
سلام وقت بخیر
لطفا نسخه سازگار با مک‌بوک، CUP: intel هم اضافه کنید.

**Comment by CodChi** [comment]
سلام، برنامه خیلی عالیه.فقط اینکه من تازه نصب کردم رو win11 ولی وقتی میزنم رو connect اجرا نمیشه.
![Screenshot 2025-05-18 220006](https://github.com/user-attachments/assets/8c6768ff-3304-4836-9df8-f8ad827fbb17)

**Reply by sajjadmrx** [reply]
سلام، چه نسخه ای دانلود و نصب کردید؟
لطفا از آخرین نسخه استفاده کنید و درصورت وجود مشکل تو دیسکورد مطرح کنید:
https://discord.gg/SpctugnwDr

**Reply by CodChi** [reply]
version 2.3.3

**Comment by masoud-ghadri** [comment]
[](url)
![Screenshot (1)](https://github.com/user-attachments/assets/9da4c733-7e5d-4a58-babe-f1d8f6d07085)
سلام بر شما من همچین اروری دارم .1- همه ی درایور ها را حذف و نصب کردم ولی کار نمیکنه . 2- آیا این DNS ها برای بازی WAR THUNDERخوب هست اگر خوب نیستن چند تا DNS خوب معرفی می کنید لطفا . ممنون میشم پاسخ بدید

**Reply by sajjadmrx** [reply]
> ![Screenshot (1)](https://private-user-images.githubusercontent.com/214910993/463367750-9da4c733-7e5d-4a58-babe-f1d8f6d07085.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTE5MTM5NjksIm5iZiI6MTc1MTkxMzY2OSwicGF0aCI6Ii8yMTQ5MTA5OTMvNDYzMzY3NzUwLTlkYTRjNzMzLTdlNWQtNGE1OC1iYWJlLWYxZDhmNmQwNzA4NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcwN1QxODQxMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hOTk1MjhiOGY1ODYxNzEyMzljNmUyYWNmOWM0YzhiZGE3MGMwMzVlNjIyODNkZTI4YmEyYzFkZTg3MjliM2Y2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.A9rMDYFh4Q5fBqwDa3-CH808FvS48NPthX2ozUT-_O4) سلام بر شما من همچین اروری دارم .1- همه ی درایور ها را حذف و نصب کردم ولی کار نمیکنه . 2- آیا این DNS ها برای بازی WAR THUNDERخوب هست اگر خوب نیستن چند تا DNS خوب معرفی می کنید لطفا . ممنون میشم پاسخ بدید

سلام، دستور داده شده رو تو ترمینال وارد کنید( با run as administrator)

**Comment by alsyundawy** [comment]
macos intel gone ?
